shared/src/business/test/createTestApplicationContext.ts

```diff
@@ -25,8 +25,8 @@ import {
 } from '../../sharedAppContext';
 import { User } from '../entities/User';
 import { abbreviateState } from '../utilities/abbreviateState';
-import { acquireLock } from '@shared/business/useCaseHelper/acquireLock';
-import { addDocketEntryForSystemGeneratedOrder } from '../useCaseHelper/addDocketEntryForSystemGeneratedOrder';
+import { acquireLock } from '@web-api/business/useCaseHelper/acquireLock';
+import { addDocketEntryForSystemGeneratedOrder } from '../../../../web-api/src/business/useCaseHelper/addDocketEntryForSystemGeneratedOrder';
```

Comment:
Hey Zach - would a package scope import work here?  In the context of a no-functional-changes change, would such a change not be as "safe" updating the relative path?

Reply:
The short answer is absolutely. The long answer is that when moving these files to web-api I am relying on vscode to update all of the imports of functions so if it was relative then it stays relative and if it was absolute it stays absolute. So if it was using "@web-api/" before then the import will continue to be "@web-api", and if it was using "../../shared/src" then it will update it to "../../web-api/src".

My preference is the @web-api/ syntax but it is just a preference. Changing all 200+ files and all of their imports to this syntax felt separate from what I was trying to achieve with this PR. Let me know if you disagree or would like to mass update all imports.